docs/pages/chat-apps/core-messaging/create-conversations.mdx

```diff
@@ -336,6 +336,46 @@ try alix.conversations. findDmbyInboxId(bo.inboxId)
 
 :::
 
+## Check membership state
```

Comment:
@mchenani what do you think of the addition of these docs? Nick suggested that perhaps we should only surface the PENDING_REMOVE status directly related to the self-removal feature?